Assets/Scripts/EOSSessionsManager.cs

```diff
@@ -290,6 +290,12 @@ public void InitActiveSession()
         {
             if (!string.IsNullOrEmpty(Name))
             {
+                if (ActiveSession != null)
+                {
+                    ActiveSession.Release();
+                    ActiveSession = null;
+                }
```

Comment:
Using null propagation, you can change this to the (arguably) clearer:
```csharp
ActiveSession?.Release();
ActiveSession = null;
```

Reply:
This is a great use of the null safe operation, good suggestion.